test/runtests.jl

```diff
@@ -91,49 +91,62 @@ sort!(testlist)
 Random.shuffle!(Oscar.get_seeded_rng(), testlist)
 
 # tests with the highest number of allocations / runtime / compilation time
-# more or less sorted by allocations
-test_large = [
-              "test/Aqua.jl",
-              "experimental/FTheoryTools/test/weierstrass.jl",
-              "test/PolyhedralGeometry/timing.jl",
-              "experimental/GITFans/test/runtests.jl",
-              "test/AlgebraicGeometry/ToricVarieties/toric_schemes.jl",
-              "test/AlgebraicGeometry/Schemes/WeilDivisor.jl",
-              "test/Rings/NumberField.jl",
-              "test/Serialization/PolynomialsSeries.jl",
-              "test/AlgebraicGeometry/Schemes/K3.jl",
-              "test/Groups/forms.jl",
-              "test/Modules/UngradedModules.jl",
-              "test/GAP/oscarinterface.jl",
-              "test/AlgebraicGeometry/Schemes/CoveredProjectiveSchemes.jl",
-              "test/AlgebraicGeometry/Schemes/CoveredScheme.jl",
-              "test/AlgebraicGeometry/Schemes/DerivedPushforward.jl",
-              "test/AlgebraicGeometry/Schemes/MorphismFromRationalFunctions.jl",
-              "experimental/QuadFormAndIsom/test/runtests.jl",
-              "experimental/GModule/test/runtests.jl",
-              "experimental/LieAlgebras/test/LieAlgebraModule-test.jl",
-              "test/Modules/ModulesGraded.jl",
-              "test/AlgebraicGeometry/Schemes/EllipticSurface.jl",
-             ]
-test_book = [
-             "test/book/test.jl",
-            ]
-
-test_subset = get(ENV, "OSCAR_TEST_SUBSET", "")
+# more or less sorted by allocations are in `long`
+# tests that should not be run for pull request CI are in `extra_large`
+# (these are run on a custom schedule only)
+test_subsets = Dict(
+              :extra_large => [
```

Comment:
The CI job sets the ENV variable `extra-long` instead of `extra_large`. I would prefer the former. Needs to be fixed in the comment a few lines above as well